### PR TITLE
Add highlighting to model solution feedback texts

### LINKED ISSUE #1118: Better highlighting of explanations when viewing questionnaire solutions
Questionnaires have configurable feedback for different answer options. One use for this is to provide students feedback on incorrect solutions; another is to explain why the correct answer is indeed correct and to elaborate on the topic of the exercise. In O1, such elaborations are sometimes fairly long and occasionally contain important information that is not explicitly written elsewhere in the course materials. Students are encouraged to read all feedback and to study example solutions as needed. 

When a student views an example solution, A+ shows the feedback/explanation for the correct answer; a simple example is shown below. That’s good, but the feedback does not stand out from the rest of the page and is easy to neglect reading it. It should be highlighted, perhaps similarly to (or better than) how O1’s ebook currently highlights feedback when a student answers a questionnaire. (I cannot remember off the top of my head if O1 has custom CSS there or not.)

image

### PR DESCRIPTION
**Pull request by Mikael-Lenander**
# Description

**What?**

Some model submission questionnaires have feedback texts. Some of these are shown also when viewing the model solution of the questionnaire. Highlights these feedback texts.

**Why?**

In some courses (O1 especially), the feedback texts contain useful information. Without highlighting, the texts are easily neglected, when reading the model answer.

**How?**

Adds css class that highlights text.

Fixes #1118

# Testing

**What type of test did you run?**

- [ ] Accessibility test using the [WAVE](https://wave.webaim.org/extension/) extension.
- [ ] Django unit tests.
- [ ] Selenium tests.
- [ ] Other test. *(Add a description below)*
- [x] Manual testing.

The highlighting only affects the style of the model solution view, not the actual questionnaire.

**Did you test the changes in**

- [ ] Chrome
- [x] Firefox
- [ ] This pull request cannot be tested in the browser.

# Is it Done?

- [ ] Reviewer has finished the code review
- [ ] After the review, the developer has made changes accordingly
- [ ] Customer/Teacher has accepted the implementation of the feature

*Clean up your git commit history before submitting the pull request!*

### CODE REVIEW

**Review by ihalaij1 — APPROVED** [review]
Looks good! Can you change the commit message to include Fixes #1118?

**Comment by Mikael-Lenander** [comment]
Also highlight hints while doing the exercise, not just model solution hints.

**Review by Mankro — CHANGES_REQUESTED** [review]
Good! Make the minor changes described below and rebase the git branch onto the upstream master. Then, we can merge.

File: assets/sass/protocols/_quiz1.scss [hunk 1]
```diff
@@ -6,6 +6,11 @@
 
 $questionnaire-focus-bg: #bce8f1;
 
+.question-hint {
+	background-color: #ffefaa;
```
[the remark this answers is not in the document]
Comment (Mankro) [reply assets/sass/protocols/_quiz1.scss:10]: In Firefox, you can install the WAVE extension to easily check if the colors pass the accessibility standards.

https://addons.mozilla.org/en-US/firefox/addon/wave-accessibility-tool/

File: assets/sass/protocols/_quiz1.scss [hunk 2]
```diff
@@ -6,6 +6,11 @@
 
 $questionnaire-focus-bg: #bce8f1;
 
+.question-hint {
```
[the remark this answers is not in the document]
Comment (Mankro) [reply assets/sass/protocols/_quiz1.scss:9]: MOOC-Grader change in its master branch:
https://github.com/apluslms/mooc-grader/commit/c697bee96202999725bd08ce0b596012a1a2e674

**Review by Mankro — APPROVED** [review]
Great! I fixed the git conflict myself and I will merge this now.

**Pull request merged**